### PR TITLE
[DAY-11] Day 11 Challenge

### PR DESCRIPTION
**Pull request by Pxyzi3c**
**PROBLEM DESCRIPTION**
In a text file, provide students with a list of names, one name per line. Ask them to create a Python program that reads the names, sorts them alphabetically, and writes the sorted names back to the file.

**HINT**
You can read the names from the file using the `.readlines()` method. This function reads each line in the file and returns a list of lines, making it easier to work with line-based data. To write the sorted names back to the file, consider using the `.writelines()` method, which allows you to write a list of strings back to the file.

### CODE REVIEW

File: backend/day-12/day-12-challenge.py
Comment (Pxyzi3c) [inline backend/day-12/day-12-challenge.py:1]: Pa disregard na lang toh yah. Sa day 12 toh kala ko may challenge eh hehe.

File: backend/day-11/day-11-challenge.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            file.writelines(f"{line}\n")
+        successfulProcess(data)
+
+def start():
+    try:
+        newContent = []
+        readFile(newContent)
+        writeFile(sorted(newContent))
+    except FileNotFoundError:
+        print("File does not exists!")
```
Comment (theadg) [inline backend/day-11/day-11-challenge.py:29]: typo on exists?

**Review by theadg — CHANGES_REQUESTED** [review]
Good shit

File: backend/day-11/day-11-challenge.py (hunk 1 above)
Comment (Pxyzi3c) [reply backend/day-11/day-11-challenge.py:29]: totally fcking not!

**Review by theadg — APPROVED** [review]

**Pull request merged**